### DISCUSSION TITLE
タグを一個も指定せずに投稿したい

### DISCUSSION THREAD

**Discussion by blono** [increments/qiita-discussions#36] [要望・アイデア・不具合]
タグは必ず 1 個以上指定されてないといけないのが現在の仕様だと思いますが、なくてもいいんじゃないかなとも思いました。
ちなみにですが、タグを 1 個も指定せずに投稿しようとした場合

> タグは0より大きい値にしてください

と表示されるのですが、メッセージに若干の違和感があります！🤔

**Comment by kskwtnk** [comment]
@blono 

ご意見ありがとうございます！

> タグは必ず 1 個以上指定されてないといけないのが現在の仕様だと思いますが、なくてもいいんじゃないかなとも思いました。

そうですね……こちらについてはだいぶ昔に実装されて以降、あまり議論になっていないため一度社内に持ち帰ってみます

また、エラーメッセージについては間違いありません……😢
もう少し分かりやすいというか、実態を表したような文章に変えようと思います！

**Comment by kskwtnk** [comment]
@blono 

こちらのDiscussionですが、以下のIssueを立てて対応しており、先ほどリリース完了しました！
ご意見くださってありがとうございます！

https://github.com/qiita-discussions/qiita-discussions/issues/41
